indico/modules/events/tracks/controllers.py

```diff
@@ -53,6 +55,18 @@ def _process_args(self):
         self.track = Track.get_one(request.view_args['track_id'])
 
 
+class RHManageTrackGroupBase(RHManageEventBase):
+    normalize_url_spec = {
+        'locators': {
+            lambda self: self.track_group
+        }
+    }
+
+    def _process_args(self):
+        RHManageEventBase._process_args(self)
+        self.track_group = TrackGroup.get_one(request.view_args['group_id'])
```

Comment:
I would call this `track_group_id` (also in the models etc) to avoid confusion with user groups